packages/ts-autocomplete/src/index.ts

```diff
@@ -190,8 +190,23 @@ function mapCompletions(
   return completions.entries
     .filter((entry) => filter({ trigger, kind: entry.kind, name: entry.name }))
     .map((entry) => {
+      /*
+      With the includeCompletionsWithInsertText option set, auto-completions
+      might have an insertText property for cases like where a property has
```

Comment:
What other cases are there and should we add tests for those as well?

Reply:
probably dashes, spaces, quotes..

Reply:
Actually, will be interesting to see some corner cases involving quotes - e.g. a collection called `foo"bar'` fails with the current autocompleter - kind of hoping it will work out of the box with the new one